ci: Add taskfiles and update GH workflows for linting Python files. (#60)
y-scope/yscope-dev-utils: exports/lint-configs/python/README.md

# Python linter configs

This directory contains standalone Python linter configuration files that double as a source of
reference settings for using the linters in your project.

> For a project with its own `pyproject.toml` file, at the time of writing, there is not a clean,
> general solution to include the standalone configuration files for different linters. In this
> scenario, the settings inside the configuration files should be copied into your `pyproject.toml`
> file, as seen in [ystdlib-py/pyproject.toml](../../ystdlib-py/pyproject.toml).

Reviewer: ```suggestion
For a project with its own `pyproject.toml` file, at the time of writing, there is no clean, general
solution for including the standalone configuration files for different linters. In this scenario,
the settings inside the configuration files should be copied into your `pyproject.toml` file, as
seen in [ystdlib-py/pyproject.toml](../../ystdlib-py/pyproject.toml).
```
Minor:
`there is not a clean` -> `there is no clean`
`to include` -> `for including`
